prover/src/composer/mod.rs

```diff
@@ -128,55 +133,41 @@ impl<E: FieldElement> DeepCompositionPoly<E> {
             i += 1;
         }
 
+        // --- merge polynomials of the composition polynomial trace ------------------------------
+        i = 0;
+        for poly in quotient_polys.into_columns() {
```

Comment:
Instead of re-using `i`, I would use `.into_columns().into_iter().enumerate()` (as `i` is not needed in subsequent loops)

Reply:
Good point, changed it now